README.md

```diff
@@ -159,6 +160,14 @@ If you have had to expose Bono and Ellis in a non-standard manner, you may need
 rake test[default.svc.cluster.local] PROXY={{Bono external IP addresss}} ELLIS={{external IP address of one of your nodes}}:30080 SIGNUP_CODE=secret
```

Comment:
As we're now setting everything up as headless services, we should update this to something like `rake test[default.svc.cluster.local] PROXY={{Bono service DNS/IP}} ELLIS={{Ellis service/IP}} SIGNUP_CODE=secret`